src/sql/workbench/browser/modal/modal.ts

```diff
@@ -267,6 +316,19 @@ export abstract class Modal extends Disposable implements IThemable {
 		this.hide();
 	}
 
+	/**
+	 * Used to close modal when a click occurs outside the modal.
+	 * This is exclusive to the Callout.
+	 * @param e The Callout modal click event
+	 */
+	private handleClickOffModal(e: any) {
```

Comment:
The type should be `MouseEvent` I believe - not any.

Reply:
You are correct.

Reply:
Please update the type and add a return type